src/shared/platforms/android/androidcommons.h

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef ANDROIDCOMMONS_H
+#define ANDROIDCOMMONS_H
+
+#include <QObject>
+
+// TODO: this needs to change!
+constexpr auto ANDROID_UTILS_CLASS = "org/mozilla/firefox/vpn/qt/VPNUtils";
+
+class AndroidCommons final : public QObject {
+  Q_OBJECT
+  Q_DISABLE_COPY_MOVE(AndroidCommons)
+
+ public:
+  static bool verifySignature(const QByteArray& publicKey,
```

Comment:
If the function moves CPP classes it should also move classes in the JVM, otherwise we this can get messy  soon - I'd prefer we it to 1 JVM class<-> 1 CPP class bindings.  
So maybe this a new class a shared package.`org.mozilla.firefox.{something}` that we can use in both flavors.